pyvista/plotting/plotting.py

```diff
@@ -1638,7 +1673,11 @@ def prepare_mapper(scalars):
         prop.SetSpecular(specular)
         prop.SetSpecularPower(specular_power)
 
-        if smooth_shading:
+        if pbr and _vtk.VTK9:
```

Comment:
Should we not raise when someone requests PBR with VTK < 9?

Reply:
Fixed